packages/newspringchurchapp/src/ui/HorizontalContentCardConnected/horizontalContentCardComponentMapper.js

```diff
@@ -13,21 +15,47 @@ const horizontalContentCardComponentMapper = ({
   ...props
 }) => {
   // map typename to the the card we want to render.
-  switch (get(props, '__typename')) {
+  const typeName = get(props, '__typename');
```

Comment:
```suggestion
  const {seriesConnection: {itemCount, itemIndex}, __typeName: typename} = props
```

and then use those variables down below instead of crazy lodash statements everywhere. it will be more readable.